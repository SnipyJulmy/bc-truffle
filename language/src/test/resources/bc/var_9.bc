tmp = 7832
v = 2
tmp
define sum(a,b) {
  auto tmp
  tmp = a + b
  return tmp
}
tmp
v
tmp
v = sum(1,2)
tmp
v
tmp
halt
