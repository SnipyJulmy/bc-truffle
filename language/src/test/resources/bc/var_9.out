7832
7832
2
7832
7832
3
7832
